src/include/block.h

```diff
@@ -522,35 +522,57 @@ struct __wt_block_disagg {
  *	The disaggregated block manager custom header
  */
 struct __wt_block_disagg_header {
+#define WT_BLOCK_DISAGG_MAGIC_BASE 0xdb
+#define WT_BLOCK_DISAGG_MAGIC_DELTA 0xdd
```

Comment:
Should the magic bytes be checked in the block disagg layer read path somewhere? Or should I do it in the block cache. Block disagg feels more appropriate.

Reply:
Right, I'll add a check for block disagg (and document it better).